src/main/resources/assets/computercraft/lua/rom/apis/colors

```diff
@@ -25,13 +25,23 @@ function combine( ... )
 end
 
 function subtract( colors, ... )
-	local r = colors
-	for n,c in ipairs( { ... } ) do
-		r = bit32.band(r, bit32.bnot(c))
-	end
-	return r
+    local r = colors
+    for n,c in ipairs( { ... } ) do
+        r = bit32.band(r, bit32.bnot(c))
+    end
+    return r
 end
 
 function test( colors, color )
     return ((bit32.band(colors, color)) == color)
 end
+
+function rgb8( r, g, b )
```

Comment:
For some reason, @Lignum decided that `setPaletteColour` could take either a 24-bit integer or three numbers between 0 and 1; this function is supposed to turn three 8-bit integers into those three numbers between 0 and 1. It's really confusing by now. Why not just stick with 24-bit RGB?